src/branch_and_bound.rs

```diff
@@ -25,12 +27,17 @@ pub const ITERATION_LIMIT: u32 = 100_000;
 /// None is returned.  Also, if no match can be found, None is returned.  The semantics may
 /// change in the future to give more information about errors encountered.
 ///
+/// If the maximum weight is exceeded, then the least valuable inputs are removed.  In so doing,
+/// minimize the number of UTXOs included in the result.
```

Comment:
This comment may be fine in the context of SRD, but I don’t think it accurately portrays the behavior in BnB. Based on your commit messages, solutions in which the maximum weight is exceeded will not be considered, but it doesn’t seem accurate that the count of UTXOs would be minimized.

Reply:
Good catch.  I did a copy and paste and forgot to update.  I'll update this accordingly.  Thanks!